src/common/asb/Asb.c

```diff
@@ -910,6 +910,11 @@ void AsbInitialize(void* log)
     FREE_MEMORY(prettyName);
     FREE_MEMORY(kernelVersion);
 
+    if (DetectSelinux(log))
+    {
+        OsConfigLogInfo(log, "AsbInitialize: SELinux present; keeping file contexts");
+    }
```

Comment:
Move after the Commodore detection and change to simply say:

"AsbInitialize: SELinux present"

Reply:
done